aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -13,81 +13,145 @@ namespace native {
 
 constexpr size_t dim_bitset_size = 64;
 
+Tensor build_index(Tensor input, int64_t flip_dim) {
```

Comment:
Can we add all these internal functions inside an anonymous namespace? Given that those names are very generic there could potentially be conflicts with other files.

See for example how it's done in https://github.com/pytorch/pytorch/blob/18edb77a28406e97d6b571f7761a50923293bf64/aten/src/ATen/native/cpu/IndexKernel.cpp#L12

So that all the `build_index`, `build_indices_loop` , `make_index_iterator` and `Indexer` are in the private namespace.

Thoughts?